### ISSUE TITLE
Installation without super user permission?

### ISSUE THREAD

**Issue by samhodge** [TheFoundryVisionmongers/nuke-ML-server#8]
Is it possible to use these tools without uid 0 being set by the user?

**Comment by ringdk** [comment]
Hello! Can you please clarify your question?

If you're referring to running Docker without being super user you can do so following these instructions: https://docs.docker.com/install/linux/linux-postinstall/#manage-docker-as-a-non-root-user

**Comment by samhodge** [comment]
In my situation I used the sudo rule several times to install CMake3 to install Docker to run Docker I know in VFX companies sudo privileges are rarely used

**Comment by ringdk** [comment]
Agreed, it's a tough one. I think we'll have to figure this one as we go. We might just have to accept that running the server requires sudo or a sudo-docker group, and hope that's OK enough for experimentation. The long term goal is to be able to take a custom trained model, then package and deploy it natively in Nuke.

**Comment by samhodge** [comment]
Yeah I know a guy that has deployed three models in C++ as an OpenFX plugin ;)
